test/functional/inspec_exec_automate_test.rb

```diff
@@ -55,11 +55,7 @@
               "job_uuid": "test123",
               "roles": ["stuff"],
               "environment": "prod",
-              "node_name": "some_node",
-              "passthrough": {
```

Comment:
Must we remove this? Chef relies pretty heavily on having the passthrough data work with the json-automate reporter, so having it explicitly tested with this structure is desirable.

Reply:
I assume it will work because of how the class inheritance with the tests, let me take care of that.